packages/kbn-ftr-common-functional-services/kibana.jsonc

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "shared-common",
+  "type": "shared-server",
```

Comment:
What makes this a "shared-server"?

Reply:
It imports server-only code such as `@kbn/core-saved-objects-server`